jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -752,7 +762,11 @@ IGN_SENSORS_DEPENDENCIES="libignition-common3-dev     \\
                           libignition-msgs3-dev       \\
                           libignition-plugin-dev  \\
                           libignition-tools-dev"
-if [[ -n "${IGN_SENSORS_MAJOR_VERSION}" && ${IGN_SENSORS_MAJOR_VERSION} -ge 3 ]]; then
+if [[ -n "${IGN_SENSORS_MAJOR_VERSION}" && ${IGN_SENSORS_MAJOR_VERSION} -ge 4 ]]; then
+  IGN_SENSORS_DEPENDENCIES="${IGN_SENSORS_DEPENDENCIES} \\
+                        libsdformat10-dev"
```

Comment:
We only have a pre-release for this, not sure if it matters.

Reply:
It will need a PR in gzdev tool, otherwise I think it should be fine.